draft-ietf-mls-protocol.md

```diff
@@ -1961,13 +1961,6 @@ message at the same time, by taking the following steps:
   applied at the leftmost unoccupied leaf, or appended to the right edge of the
   tree if all leaves are occupied.
 
-* Create an initial, partial GroupInfo object reflecting the following values:
-  * Group ID: The group ID for the group
-  * Epoch: The epoch ID for the next epoch
-  * Tree: The group's ratchet tree after the commit has been applied
-  * Prior confirmed transcript hash: The confirmed transcript hash for the
-    current state of the group (not the provisional state)
-
 * Create a DirectPath using the new tree (which includes any new members).  The
   GroupContext for this operation uses the `group_id`, `epoch`, `tree`, and
   `prior_confirmed_transcript_hash` values in the initial GroupInfo object.
```

Comment:
This change leaves a reference to a GroupInfo object that is not mentioned before. If I'm not mistaken, I think this can be rewritten as:

> Create a DirectPath using the new tree (which includes any new members). The GroupContext for this operation uses the `group_id`, `epoch`, and `tree` in the provisional GroupContext and the `confirmed_transcript_hash` for the current state of the group (not the provisional state).

On this note, I'm not entirely sure I understand why an entire GroupContext (specifically the `group_id`, `epoch`, and `confirmed_transcript_hash`) is needed to compute a DirectPath...